code/modules/surgery/anesthesia_machine.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/anesthetic_machine
-	name = "Anesthetic Tank Holder"
-	desc = "A wheeled machine that can hold a tank and distribute the gas inside using a breath mask."
+	name = "Tank Holder"
```

Comment:
tank holder is a bit generic, maybe something that references it's use in a medical setting or that it's tank related
try looking for a real life thing to name it after perhaps

Reply:
ok